CriteoPublisherSdk/Tests/UnitTests/Standalone/CRBannerViewTests.m

```diff
@@ -248,9 +248,23 @@ - (void)testCreateWebViewWithConfiguration {
   [self cr_waitShortlyForExpectations:@[ openInBrowserExpectation ]];
 }
 
-#pragma inhouseSpecificTests
-
 - (void)testLoadingSuccess {
+  MockWKWebView *mockWebView = [MockWKWebView new];
+  NSString *displayURL =
+      @"https://rdi.eu.criteo.com/delivery/r/ajs.php?did=5c98e9d9c574a3589f8e9465fce67b00&u=%7Cx8O2jgV2RMISbZvm2b09FrpmynuoN27jeqtp1aMfZdU%3D%7C&c1=oP5_e7JVVt0EkjVehxP6aIOIWS-fm2fzhyMXUboeuR1zkGydE3HlloxT1QAbHNNgeH7t9e1IR6mv0biMxm46ZSFdAXZXreJVeP6QwU8IPLUsA32HNafhqgpnKTwmx9RrrJm4CS5Wqj07vNY7UTgDei8AWqc5CGPT2wm7W02JRvgN2kA-oWbWifmmm6EPpqVZijDHDzXwaNgzrfsaEodEmYAjFepGF0mdElHoFUCPKuOtc7mUQijLG0BSS9RhwrCTcAv42KkEQ359Et_eDnQcSt9OAF3bL64QIvLQxt2ekYFNuv3zng03qL0DIHS2bDJwRb3ieUlvZCWHI49OqM5PqoGDpSzdhdwfTE18L6cOOVKqPQ0dPofN4dkSs9IbVGiYlPnjfibL88PwTspYvki2svidSDIa2agQMHVgEof8YY4x4VgPjA8XY-s93ttw_i-RN3lcQn2mGEp6FYmRsyjFEDxHgGfJ0j6U";
```

Comment:
This URL is really huge, and I don't think that using a real one brings a lot to this test. What do you think about using a dummy one instead ?

Reply:
You could use: `https://rdi.eu.criteo.com/delivery/rtb/demo/ajs?zoneid=1417086&width=300&height=250&ibva=0` for banners, and  `https://rdi.eu.criteo.com/delivery/rtb/demo/ajs?zoneid=1417086&width=393&height=759&ibva=1&uaCap=5`for interstitials.

Reply:
Done, refactored the whole test as this URL was referenced 6 times